drivers/hud_spm_report/app/models/hud_spm_report/generators/fy2023/measure_two.rb

```diff
@@ -20,13 +20,13 @@ def self.client_class
     def self.table_descriptions
       {
         'Measure 2' => 'The Extent to which Persons Who Exit Homelessness to Permanent Housing Destinations Return to Homelessness within 6, 12, and 24 months',
-        '2a and 2b' => 'The Extent to which Persons Who Exit Homelessness to Permanent Housing Destinations Return to Homelessness within 6, 12, and 24 months.',
+        # '2a and 2b' => 'The Extent to which Persons Who Exit Homelessness to Permanent Housing Destinations Return to Homelessness within 6, 12, and 24 months.',
```

Comment:
Didn't you just update the exiting data to use `2a and 2b`?  I'm not following the logic in determining which it should or should not be and where.

Reply:
Table descriptions is used by the view to add titles to the view. Because M2 only has 1 table, we don't need both, and since the table had been incorrectly named, this second line didn't appear -- but having fixed the table name, the result was confusing.